cpp/benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+
+project(CUSPATIAL_BENCHS LANGUAGES C CXX CUDA)
+
+if(NOT CMAKE_CUDA_COMPILER)
+  message(SEND_ERROR "CMake cannot locate a CUDA compiler")
+endif()
+
+###################################################################################################
+# - common test utils -----------------------------------------------------------------------------
+
+find_library(CUDFTESTUTIL_LIBRARY "libcudftestutil.a"
+             HINTS "$ENV{CUDF_ROOT}/lib")
+
+message(STATUS "CUDFTESTUTIL: CUDFTESTUTIL_LIBRARY set to ${CUDFTESTUTIL_LIBRARY}")
+message(STATUS "CUDFTESTUTIL: CUDF_TEST_INCLUDE set to ${CUDF_TEST_INCLUDE}")
+
+add_library(cudftestutil STATIC IMPORTED ${CUDFTESTUTIL_LIBRARY})
+if (CUDF_TEST_INCLUDE AND CUDFTESTUTIL_LIBRARY)
+    set_target_properties(cudftestutil PROPERTIES IMPORTED_LOCATION ${CUDFTESTUTIL_LIBRARY})
```

Comment:
Might need position independent code option here (@trxcllnt  did that work for you?)

Reply:
It did not. I couldn't replicate with gcc 7.5 in rapids-compose, but I think CI might be using gcc 7.3.

Reply:
Going to keep this as-is until we can determine what change is needed, if any.